pandas/core/frame.py

```diff
@@ -4615,6 +4615,7 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None,
                 else:
                     raise TypeError('must specify how or thresh')
 
+            #result = self._take(mask.to_numpy().nonzero()[0], axis=axis)
```

Comment:
This is the pre-existing implementation that can be located in git history. It's not needed here.